tests/index.html

```diff
@@ -107,7 +104,7 @@
 	var gameWidth = window.innerWidth-1-toolbarSz;
 	var gameHeight = window.innerHeight-1;
 	var contra;
-	var padd = 20;
+	var padd = 20*gameWidth/1087;
```

Comment:
How did you get to this value?
"padd" is also used to determine the padding of images in each of the 8 positions